association-de-science-regionale-de-langue-francaise.csl

```diff
@@ -0,0 +1,482 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="fr-CA">
```

Comment:
Is this style for Canadian French (locale code "fr-CA") or 'regular' French (locale code "fr-FR")?